new_samples/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"github.com/uber-common/cadence-samples/new_samples/worker"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func main() {
+	worker.StartWorker()
+
+	done := make(chan os.Signal, 1)
+	signal.Notify(done, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
```suggestion
	signal.Notify(done, syscall.SIGINT)
```
unless you found evidence that term is necessary, I wouldn't include it.  no need to contribute to the misinformation.

Reply:
accepted. Thanks Steve.